src/domdiv/fonts/README.md

```diff
@@ -0,0 +1,15 @@
+## Fonts
+
+I believe I cannot distribute one font (Minion Pro) domdiv uses as they are owned by Adobe with a License that I understand to disallow redistribution.
```

Comment:
```suggestion
I believe I cannot distribute one font (Minion Pro) domdiv uses as they are owned by Adobe with a License that I understand to disallow redistribution. However, you can download the [3 font files from a third party here](https://www.dropbox.com/s/tsqk69mayoa3pfz/MinionPro-ForDominionTabs.zip?dl=1).
```

Reply:
Easier to just grab a zip file with the 3 fonts formatted as needed than worry about conversion etc.